backend/api/app/application/services/user_service.py

```diff
@@ -50,6 +78,9 @@ def login_user(self, email: str, password: str, ip_address: str):
         if not user.check_password(password):
             raise ValueError({"message": "Password is incorrect.", "status": 401})
         
+        if user.is_deleted == True:
+            raise ValueError({"message":"User does not exist.", "status":404})
```

Comment:
Keep the message as the same as in the row 76, since the deleted users should be treated as they never existed.